api.go

```diff
@@ -0,0 +1,17 @@
+package apidApigeeSync
+
+import (
+	"github.com/30x/apid-core"
+	"net/http"
+)
+
+const tokenEndpoint = "/accesstoken"
+
+func InitAPI(services apid.Services) {
+	services.API().HandleFunc(tokenEndpoint, getAccessToken).Methods("GET")
+}
+
+func getAccessToken(w http.ResponseWriter, r *http.Request) {
+	<-tokenUpdatedApiNotificationChan
+	w.Write([]byte(apidTokenManager.getBearerToken()))
+}
```

Comment:
The behavior is: When call "/accesstoken" endpoint, it will hang until a new token retrieved.
Is this the expected behavior?